docs/guides/v2_v3_guide/v2_to_v3_guide.md

```diff
@@ -62,24 +62,29 @@ GatewayIntents = GatewayIntents.AllUnprivileged | GatewayIntents.GuildMembers |
 
 [developer portal]: https://discord.com/developers/
 
-### ReactionAdded Event
+### UserLeft event
+
+UserLeft has been changed to have the ` SocketUser ` and ` SocketGuild ` parameters instead of a ` SocketGuildUser ` parameter. 
+Because of this, guild-only user data cannot be retrieved from this user anymore, as this user is not part of the guild.
+
+### ReactionAdded event
 
 The reaction added event has been changed to have both parameters cacheable.
 This allows you to download the channel and message if they aren't cached instead of them being null.
 
-### UserIsTyping Event
+### UserIsTyping event
 
 The user is typing event has been changed to have both parameters cacheable.
 This allows you to download the user and channel if they aren't cached instead of them being null.
 
 ### Presence
 
-There is a new event called `PresenceUpdated` that is called when a user's presence changes,
-instead of `GuildMemberUpdated` or `UserUpdated`.
+There is a new event called ` PresenceUpdated ` that is called when a user's presence changes,
+instead of ` GuildMemberUpdated ` or ` UserUpdated `.
```

Comment:
> ` PresenceUpdated `

Is there any reason why you have decided to add an extra space between the graves and the term? 

This is inconsistent behaviour with: 

https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/074ed34215f314be800d96f52008a0268b86236d/docs/guides/v2_v3_guide/v2_to_v3_guide.md?plain=1#L16

and other files such as:

https://github.com/Discord-Net-Labs/Discord.Net-Labs/blob/074ed34215f314be800d96f52008a0268b86236d/docs/guides/voice/sending-voice.md?plain=1#L14

Reply:
Done subconsciously, thank you for noticing.